WordPress/src/main/res/values/strings.xml

```diff
@@ -353,6 +353,12 @@
     <string name="snackbar_conflict_web_version_discarded">Web version discarded</string>
     <string name="snackbar_conflict_undo">Undo</string>
 
+    <!-- post autosave conflict dialog -->
+    <string name="dialog_confirm_autosave_title">More Recent Version Conflict</string>
```

Comment:
I'd personally avoid the word "Conflict" even here. Eg. Calypso uses "more recent revision".